.travis.yml

```diff
@@ -3,4 +3,4 @@ script: "./script/cibuild"
 gemfile: "this/does/not/exist"
 env: GITHUB_API_TOKEN=36a1b2a815b98d755528fa6e09b845965fe1e046 # token of user: some-public-token without any permissions to avoid api rate limit
 rvm:
-  - "1.8.7"
+  - "2.1.5"
```

Comment:
@isaacsu only we test versions of ruby that are on OSX by default, which means that we need `1.8.7` and `2.0.0`.

Can you update the `.travis.yml` to look like the one that [puppet-template](https://github.com/boxen/puppet-template/blob/master/.travis.yml) uses?